app/src/main/res/layout/activity_feedback.xml

```diff
@@ -0,0 +1,133 @@
+
+<android.support.v4.widget.DrawerLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools" android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="com.kanshu.kanshu.FeedbackActivity"
+    android:id="@+id/main_layout"
+    android:fitsSystemWindows="true"
+    android:background="#F5F5F5">
+    <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical">
+    <android.support.v7.widget.Toolbar
+        android:id="@+id/toolbar"
+        android:layout_height="wrap_content"
+        android:layout_width="match_parent"
+        android:elevation="0dp"
+        android:paddingTop="25dp"
+        android:minHeight="?attr/actionBarSize"
+        android:background="?attr/colorPrimary"/>
+        <TextView
+            android:layout_width="match_parent"
+            android:layout_height="45dp"
+            android:paddingLeft="40dp"
+            android:textColor="@color/navbar_off_white"
+            android:gravity="center_vertical"
+            android:text="@string/feedback_send"
+            android:background="@drawable/feedback_header_bg"/>
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="30dp"
+        android:layout_marginTop="30dp">
+
+        <TextView android:text="@string/feedback_from" android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textColor="@color/article_dart_grey"
+        android:id="@+id/textFrom" />
+
+        <EditText
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/feedback_default_email"
```

Comment:
android:text should be android:hint so that the text doesn't have to be deleted when the user clicks on it